frontend/packages/helm-plugin/integration-tests/support/pages/helm/helm-details-page.ts

```diff
@@ -36,4 +36,42 @@ export const helmDetailsPage = {
     cy.get('form strong').should('have.text', releaseName);
     cy.get(helmPO.uninstallHelmRelease.releaseName).type(releaseName);
   },
+  checkHelmTab: (name: string) => {
+    cy.get(`a[data-test-id="horizontal-link-${name}"]`).should('exist');
+  },
+  selectHelmTab: (name: string) => {
+    cy.get(`a[data-test-id="horizontal-link-${name}"]`)
+      .should('exist')
+      .click();
+  },
+  verifyHelmActionsDropdown: () => cy.byLegacyTestID('dropdown-button').should('be.visible'),
+  clickHelmActionButton: () => cy.byLegacyTestID('dropdown-button').click(),
+  verifyActionsInCreateMenu: () => {
+    cy.get('.pf-c-dropdown__menu-item')
```

Comment:
better to use `data-test-id` to get the dropdown menu item instead of class.

Reply:
Updated the PR